package-lock.json

```diff
@@ -3979,6 +3979,14 @@
       "integrity": "sha1-w2GT3Tzhwu7SrbfIAtu8d6gbHA8=",
       "dev": true
     },
+    "monotone-convex-hull-2d": {
```

Comment:
> It is. If only to detect when it's missing (like in #400).

Yeah right

> also in case it differs from what the package.json says (like here

Here, the commit put it back in sync with the current state of package.json. But ok, I'll make sure #413 is merged before this one, that'll fix everything and the diff will be pretty :-)